src/config.ts

```diff
@@ -116,7 +135,8 @@ export const appConfig: IApplicationConfig = {
   clientLoginRedirectionUrl: CLIENT_REDIRECTION_URL,
   loginPath: "/login",
   metadataPath: "/metadata",
-  sloPath: "/slo"
+  sloPath: "/slo",
+  startupIdpsMetadata: STARTUP_IDPS_METADATA
```

Comment:
can you please document this var into th readme ?

Reply:
Fixed in 53626f2